portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,8 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+function getName(name) {
```

Comment:
The lint check isn't smart enough to know that this function is referenced in the HTML, so it's showing an error when it's actually totally fine. You can tell it to stop error-ing on the function being unused by adding this comment at the end of the line: `// eslint-disable-line`

It would look like:
```javascript
function getName(name) { // eslint-disable-line
```

Feel free to do the same for `addRandomGreeting`